SYCL/ESIMD/accessor_load_store.cpp

```diff
@@ -48,10 +49,15 @@ template <typename T> struct char_to_int {
 
 template <typename T> bool test(queue q, size_t size) {
   std::cout << "Testing T=" << typeid(T).name() << "...\n";
+  esimd_test::input_gen::Generator<T> gen(
```

Comment:
No seed-ing for reproducibility?

Reply:
I wanted usage of this generator as simple as possible. Maybe custom seeding can be added later.